docs/core/testing/mstest-analyzers/mstest0001.md

```diff
@@ -47,12 +47,15 @@ Do not suppress a warning from this rule. Many libraries can benefit from a mass
 
 ## Suppress a warning
 
-Violations to this rule cannot be suppressed inline.
+Because this rule is reported on compilation-level, and not reported in a `.cs` or `.vb` source file, violations to this rule cannot be suppressed inline nor via `.editorconfig`.
```

Comment:
```suggestion
Because this rule is reported at the compilation level and not in a *.cs* or *.vb* source file, you can't suppress violations to this rule inline or via an `.editorconfig` file.
```